routes/scrape_routes.js

```diff
@@ -0,0 +1,71 @@
+var request = require("request");
+var db = require("../models");
+var cheerio = require("cheerio");
+
+module.exports = function(app) {
+	app.get("/scrape/:subreddit?", function(req, res) {
+		var subreddit = req.params.subreddit;
+		if(!req.params.subreddit) {
+			subreddit = "all"
+		}
+	  // Make a request for the news section of ycombinator
+	  request("https://www.reddit.com/r/"+subreddit, function(error, response, html) {
+	    // Load the html body from request into cheerio
+	    var $ = cheerio.load(html);
+	    // For each element with a "title" class
+	    var page = [];
+	    var articles = [];
+	    var count = $('div#siteTable > div.link').length;
+	    var site = $('div#siteTable > div.link');
+	    $('div#siteTable > div.link').each(function( index ) {
+	    	var article = {};
+		    article.title = $(this).find('p.title > a.title').text().trim();
+		    article.titleLink = $(this).find('p.title > a.title').attr("href").trim();
+		    if(article.titleLink[0] === "/") {
```

Comment:
I would imploy a better check here, a quick google search would reveal a good regex pattern that would better verify an href.